### PR TITLE
Format logs

### PR DESCRIPTION
**Pull request by mars-lan**
### 🤔 Why?

The current log format lacks timestamp & log level info. It also logs at DEBUG level for ALL modules.

### 🤓 What?

- Share a single logger for all metaphor modules
- Prefix log entries with timestamp & log level
- Only output Metaphor's DEBUG logs to file

### 🧪 Tested?

Before:
```
Executing bigquery connector with config file /Users/marslan/Desktop/local/bigquery.yml
Starting extractor BigQueryExtractor at 2022-10-28 05:10:13.608983
Fetching metadata from BigQuery project metaphor-data
```

After:
```
2022-10-28 05:12:53:INFO:metaphor:Executing bigquery connector with config file /Users/marslan/Desktop/local/bigquery.yml
2022-10-28 05:12:53:INFO:metaphor:Starting extractor BigQueryExtractor at 2022-10-28 05:12:53.947320
2022-10-28 05:12:53:INFO:metaphor:Fetching metadata from BigQuery project metaphor-data
```

### CODE REVIEW

File: metaphor/common/logger.py [hunk 1]
```diff
@@ -3,14 +3,23 @@
 
 _, LOG_FILE = tempfile.mkstemp(suffix=".log")
 
-logging.basicConfig(level=logging.DEBUG, filename=LOG_FILE, filemode="a")
+formatter = logging.Formatter(
+    fmt="%(asctime)s:%(levelname)s:%(name)s:%(message)s", datefmt="%Y-%m-%d %H:%M:%S"
```
Comment (alyiwang) [inline metaphor/common/logger.py:7]: Do we need this `(name)` in every log? It's always `metaphor`

**Review by alyiwang — COMMENTED** [review]
Otherwise, LGTM

File: metaphor/common/logger.py (hunk 1 above)
Comment (mars-lan) [reply metaphor/common/logger.py:7]: It helps distinguish logs generated by Metaphor's code vs third party libraries.

Comment (alyiwang) [reply metaphor/common/logger.py:7]: sounds good

**Review by alyiwang — APPROVED** [review]

**Pull request merged**